packages/docusaurus-theme-classic/src/theme/DocCardList/index.tsx

```diff
@@ -27,9 +27,9 @@ export default function DocCardList({items, className}: Props): JSX.Element {
   return (
     <div className={clsx('row', className)}>
       {filterItems(items).map((item, index) => (
-        <article key={index} className="col col--6 margin-bottom--lg">
```

Comment:
Why change the `article` with `section` here? Can't we keep the `article` as repeatable item? I would change the parent `div` with `row` class to `section` tag in that case.

Reply:
So, articles within sections? I thought sections should be descendants of articles, though. I'm fine with making the card and the list both `article`.

Reply:
Ah, sorry, misread. Makes sense.